src/components/Register/CreateRestaurant/CreateRestaurant.js

```diff
@@ -35,33 +38,35 @@ const CreateRestaurant = props => {
 				}
 			}
 		}
-		const response = await axios.post('/api/register', {restaurant})
-			props.updateRestaurant(restaurant)
-			activateRedirect(true)
+		try {
+			console.log('submitting restaurant', restaurant)
+			const response = await axios.post('/api/register', {restaurant})
+			console.log('submission successful',response)
+		} catch (error) {
+			console.log('submission not successful')
+			console.log(error)
+		}
+		props.updateRestaurant(restaurant)
 	}
-
-
 	return (
-		redirect
-		? //after form is complete, redirect to home
-			<Redirect to="/home"></Redirect>
-		: //if restaurant object is empty, capture basic info
+		//if restaurant object is empty, capture basic info
 		!restaurantObj.email
 		? 
-			<form onSubmit={captureBasicInfo}>
-				<input value={name} onChange={(e)=>updateName(e.target.value)} placeholder="restaurant name" name="name"/>
-				<input value={owner} onChange={(e)=>{updateOwner(e.target.value)}} placeholder="owner name" name="owner"/>
-				<input value={email} onChange={(e)=>{updateEmail(e.target.value)}} placeholder="email" name="email"/>
-				<button>Next</button>
-			</form>
+			<>
```

Comment:
"e" is passed implicitly if you just put function definition. This one is a pretty big deal to employes since there are so many instances between 56-67